zio/parquetio/reader.go

```diff
@@ -79,25 +50,58 @@ const (
 	list
 )
 
-type parquetColumn struct {
-	goName   string
-	typ      HandledType
-	listType *parquetColumn
-	name     string
-}
+func lookupPrimitiveType(typ *parquet.Type, cType *parquet.ConvertedType) (HandledType, bool) {
```

Comment:
For debugging purposes it would be nice to have this method return and error instead and when you hit the default in your cases do: `fmt.Errorf("cannot convert type: %s", typ)`